chapter-customizing.asciidoc

```diff
@@ -1462,7 +1463,7 @@ provides a solid foundation that makes it easier to predict and manage
 the plugin goals which will be executed in a given build. In Maven 1,
 plugin goals related to one another directly; in Maven 2, plugin goals
 relate to a set of common lifecycle stages. While it is certainly
-valid to execute a plugin goal directly from the command-line as we
+valid to execute a plugin goal directly from the command line as we
```

Comment:
"command-line" only needs to be hyphenated when it's being used as an adjective.